docs/manager.md

```diff
@@ -217,7 +222,7 @@ If the `ps aux | grep qemu-system-x86_64` give you something like this
 darko      13913  0.0  0.0      0     0 pts/2    Z+   20:17   0:00 [qemu-system-x86] <defunct>
 ```
 
-means that the a QEMU virtual machine that is currently defunct, meaning that it is no longer running. More precisely, the defunct process in the output is also known as a ["zombie" process](https://en.wikipedia.org/wiki/Zombie_process).
+means that the QEMU virtual machine that is currently defunct, meaning that it is no longer running. More precisely, the defunct process in the output is also known as a [&#34;zombie&#34; process](https://en.wikipedia.org/wiki/Zombie_process).
```

Comment:
What is the particular reason for these escape characters? Markdown should be capable of using `"` chars directly in the text, right?

Reply:
Yes, direct quotation marks work in Markdown. It seems that VS Code automatically adds escape characters, possibly for compatibility with different HTML rendering systems and to prevent vulnerabilities like cross-site scripting. Since GitHub's Markdown renderer already handles these issues, would you prefer that I revert the escape characters?

Reply:
Yes, please - revert.

Reply:
This reverts the changes manually.